### ISSUE TITLE
`no_qgoodwindow` flag: `'return': cannot convert 'int' to 'QFlags<Qt::AlignmentFlag>'`

### ISSUE THREAD

**Issue by kresimirko** [antonypro/QGoodWindow#24]
Hello again. Here's another issue I've encountered while trying out the `no_qgoodwindow` flag:

![image](https://user-images.githubusercontent.com/104278083/215892717-2ba019e3-7c64-4c1b-9a6e-404d38ed8282.png)

```
[...]\QGoodWindow-master\QGoodWindow\QGoodCentralWidget\src\qgoodcentralwidget.cpp:435: error: C2440: 'return': cannot convert from 'int' to 'QFlags<Qt::AlignmentFlag>'
..\..\QGoodCentralWidget\src\qgoodcentralwidget.cpp(435): error C2440: 'return': cannot convert from 'int' to 'QFlags<Qt::AlignmentFlag>'
..\..\QGoodCentralWidget\src\qgoodcentralwidget.cpp(435): note: No constructor could take the source type, or constructor overload resolution was ambiguous
```

Occurs here at line 435: https://github.com/antonypro/QGoodWindow/blob/d0a17a62ef81a08a28d929e07bfce3e1444674a0/QGoodWindow/QGoodCentralWidget/src/qgoodcentralwidget.cpp#L430-L437

Encountered while trying to build GoodShowCase & GoodPlayground. I didn't try this out on the other examples.

**Comment by antonypro** [comment]
@kresimirko: This problem is solved by replacing `0` with `Qt::Alignment(0)`.

The next version of the library will contain the fix.

Thanks for reporting!

**Comment by antonypro** [comment]
@kresimirko: This bug was solved on version 2.3.
